lib/SectionsManager.jsm

```diff
@@ -35,6 +35,7 @@ const BUILT_IN_SECTIONS = {
       link: {
         href: "https://getpocket.com/firefox/new_tab_learn_more",
         message: {id: "newtab-pocket-how-it-works"},
+        id: "pocket_how_it_works"
```

Comment:
nit: need to end with comma to make lint happy

Reply:
Hmm.. So this link object is used in both newtab and about:preferences… I guess restoring `id` happens to fix it now, but I wonder how we'll want to fix it when migrating preferences strings.

Reply:
I suppose instead of just reusing the link object, a separate link object could be set in the `pref` object above.

Reply:
so this should be good now, when migrating preferences strings, we can create a separate link object inside Pref